hibernate-core/src/main/java/org/hibernate/query/sqm/sql/internal/StandardSqmSelectTranslator.java

```diff
@@ -86,7 +85,9 @@
 	// prepare for 10 root selections to avoid list growth in most cases
 	private final List<DomainResult> domainResults = CollectionHelper.arrayList( 10 );
 
-	private GraphImplementor<?> currentJpaGraphNode;
+	private final EntityGraphNavigator entityGraphNavigator;
+
+	private int fetchDepth;
```

Comment:
I moved the field here for best practice. Also, it seems we don't need to set default value here.